### PR TITLE
Fixes #813

### LINKED ISSUE #813: Special prey interactions affecting wrong belly
#### Steps to reproduce
- Create a second organ
- Give the organ any chance to absorb
- Prey in that organ uses the Resist verb until this chance triggers
- First organ gets set to Absorb rather than the organ the prey was in

#### Code Revision
- cda702a5ee2a28c08723ba6397219c1bb609bcaa (I cloned as zip from this point, deleted the old folders and performed a clean compile)

### PR DESCRIPTION
**Pull request by Cameron653**
Fixes #813 
**Full credit to Aronai for the bugfix**. 

I couldn't figure it out for the life of me but then they pointed out that I was simply over complicating it and gave me this bugfix

Tested it and I can confirm this fixes the problem.

Also fixes two typos

### CODE REVIEW

File: code/modules/vore/eating/belly_vr.dm [hunk 1]
```diff
@@ -427,18 +427,14 @@
 		else if(prob(absorbchance)) //After that, let's have it run the absorb chance.
 			R << "<span class='warning'>In responce to your struggling, \the [name] begins to get more active...</span>"
```
Comment (Screemonster) [inline code/modules/vore/eating/belly_vr.dm]: Typo. Should be "response", not "responce".

File: code/modules/vore/eating/belly_vr.dm [hunk 2]
```diff
@@ -427,18 +427,14 @@
...
 			owner << "<span class='warning'>You feel your [name] beginning to become active!</span>"
-			for(var/K in owner.vore_organs)
-				var/datum/belly/B = owner.vore_organs[K]
-				B.digest_mode = DM_ABSORB
-				return
+			digest_mode = DM_ABSORB
+			return
 
 		else if(prob(digestchance)) //Finally, let's see if it should run the digest chance.
 			R << "<span class='warning'>In responce to your struggling, \the [name] begins to get more active...</span>"
```
Comment (Screemonster) [inline code/modules/vore/eating/belly_vr.dm]: Same typo as above here.

**Comment by Screemonster** [comment]
Spotted a couple of typos, "response" misspelled as "responce"

**Comment by Cameron653** [comment]
Alright, I'll go ahead and fix those when I'm home

**Comment by Cameron653** [comment]
Fixed the typos.

**Pull request merged**